dependencies.yaml

```diff
@@ -67,7 +67,7 @@ libevse-security:
 # OCPP
 libocpp:
   git: https://github.com/EVerest/libocpp.git
-  git_tag: 9836ac4766e99a79555adb15c3001c8704f8b7a7
+  git_tag: bugfix/meter-value-pricing-trigger
```

Comment:
This should be set back to the correct version as soon as it the bugfix (https://github.com/EVerest/libocpp/pull/899) is merged.